posts/2020-10-23-beta-20.0.0.12.adoc

```diff
@@ -0,0 +1,452 @@
+---
+layout: post
+title: "MicroProfile Updates, New Jakarta EE 9 Packages and more in Open Liberty 20.0.0.12-beta"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/austin0
+author_github: https://github.com/austin0
+seo-title: MicroProfile Updates, New Jakarta EE 9 Packages and more in Open Liberty 20.0.0.12-beta - OpenLiberty.io
+seo-description: OpenLiberty updates for a number of commonly used MicroProfile features and four new Jakarta EE 9 packages, now available in Open Liberty 20.0.0.12-beta.
+blog_description: Updates for a number of commonly used MicroProfile features and four new Jakarta EE 9 packages.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= MicroProfile Updates, New Jakarta EE 9 Packages and more in Open Liberty 20.0.0.12-beta
+Austin Bailey <https://github.com/austin0>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+Open Liberty 20.0.0.12-beta comes complete with a range of previews for MicroProfile updates, including support for MicroProfile 4.0 and accompanying updates to MicroProfile Config (2.0) and MicroProfile JWT (1.2). Also available for preview are a multitude of Jakarta EE 9 features amongst which are changes to the Jakarta RESTful Web Services specification and the addition of message-driven bean support.
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try either package, <<feedback, let us know what you think>>.
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features:
+
+* <<mp4, Support for MicroProfile 4.0>>
+* <<conf, Support for MicroProfile Config 2.0>>
+* <<jwt, Support for MicroProfile JWT 1.2>>
+* <<lra, Support for MicroProfile LRA>>
+* <<java15, Support for Java SE 15>>
+
+[#mp4]
+=== Support for MicroProfile 4.0
+
+The Open Liberty 20.0.0.12-beta provides most feature updates for MicroProfile 4.0-M1, this includes updates to MicroProfile Config, Fault Tolerance, Metrics, Health, Open Tracing, Open API and JWT, detailed below. While MicroProfile Rest Client is part of the MicroProfile 4.0-M1 specification, this feature narrowly missed the beta train and is scheduled for delivery in milestone 2.
+
+MicroProfile 4.0 brings the following feature updates to Open Liberty:
+
+* Config 2.0
+* Fault Tolerance 3.0
+* Metrics 3.0
+* Health 3.0
+* Open API 2.0
+* JWT 1.2
+* Open Tracing 2.0
+
+To enable the new beta features in your app, add them to your `pom.xml`:
```

Comment:
Change this sentence to "To enable all the available MicroProfile 4.0 features at once, just add our MicroProfile 4.0 convenience feature to your `server.xml`:"

Reply:
Changed.